builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -395,6 +395,7 @@ local function parse_config_file(read_all, parse_mods)
 
 				table.insert(settings, {
 					name = mod.name,
+					readable_name = mod.title,
```

Comment:
maybe call it `title` here too for consistency?

Reply:
This isn't a new property - `readable_name` is what it is already called elsewhere.
This is the All Settings menu's category names. Normally categories are always "readable", so just the `name` property is used, but here it's making the categories for game's settingtypes.txt, so there is both the technical `name` and the `readable_name`.

Reply:
You could argue all the other categories should be changed to use `readable_name`.

Reply:
Ok then